cli/waiter/action.py

```diff
@@ -90,6 +90,18 @@ def check_service_status():
             print(f'Service is currently {format_status(service_status)}.')
     return succeeded
 
+def check_ssl(token_name, timeout_seconds):
+    """Returns true if a request to the token's DNS name doesn't encounter an SSL error"""
+    timeout_millis = timeout_seconds * 1000
+    try:
+        http_util.__get(f"https://{token_name}", read_timeout=timeout_millis)
```

Comment:
Should we log the response for debugging purposes?

Reply:
What would the contents of the response help with?  We only care about whether the connection can be established and a ssl handshake completed
Logging the response would only happen when the command exits successfully
When the command fails it never receives a response (because it throws trying to get it)